submissions/ZainabAbdussalam-Osimade/HTML/Product.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<head>
+      <title>www.FamFashionHub.com</title>
+      <body>
+            <h1>Fam Fashion Hub</h1>
+            
+            <p>Jackets <br/> Hooded tops <br/> Round necks <br/> Waist Coats </p>
+            
+            <br>
+  
+            <img src="Images/1 image.jpg" width="200px"/>
+
+            <p>
+                 <b>Fam Collection Women's Packable Puffer Jacket with Hood </b> <br/> <b> #35,000 </b>
+            </p>
+            
+            <p> Available in Wine, Camo, and Black </p>
+  
+            <p>
+            <b>Feature</b>
+            </p>
+
+            <p>Giving you the desired warmth. You can go anywhere, do anything with this puffer jacket. <br/> This on-trend coat compresses and packs in its own pocket/bag, making it the perfect traveling companion. </p>
+    
+            <p>Stand collar with fleece lining; detachable hood <br/> Zip-front closure <br/> Two front zipper pockets <br/> Allover quilted pattern <br/> Wind resistant <br/> Compresses and folds into its own pocket for easy packing <br/> Solid shell: 100% Nylon; Lining: 100% Polyester; Print shell and lining: 100% Polyester <br/> Machine washable </p>
+            
```

Comment:
Your code is not properly formatted. You do not need to leave a line after every  line of code, kindly remove whitespaces. The body tag is not properly formatted.